contents/security/crossSiteScripting/crossSiteScripting.md

```diff
@@ -75,24 +90,79 @@ However, that also means that hackers are able to also include scripts in their
 
 `https://www.ecard.com/view-ecard.php?content=Happy%20Holidays<script>...</script>`
 
-## Preventing Cross Site Scripting
+## Well-Known XSS Incidents
+
+**MySpace Worm**
+
+In 2005, MySpace allowed users to customise their profiles using HTML code. This allowed for diversity of profiles but also allowed Samy Kamkar to find an XSS vulnerability. 
+
+Samy Kamkar wrote a script which made people who visited his profile send him a friend request, and list Samy’s in their own profile’s “My Heroes” section. Not only that, he also programmed the script to copy itself onto the visitor's profile. Within a day, he had 1 million friend requests. 
+
+This caused MySpace to take the site offline to figure out what was going on and to purge the worm. 
+
+You can read the detailed article <a href="https://motherboard.vice.com/en_us/article/wnjwb4/the-myspace-worm-that-changed-the-internet-forever" target="_blank">here</a>.
+
+**Self-Retweeting Tweet**
+
+In 2014, a tweet became the world's first retweeting tweet. Twitter itself has security measures against XSS and is thus unaffected. However, users on TweetDeck (Twitter social media dashboard application) that came across this tweet will automatically retweet this tweet:
+
+```html
+<script class="xss">
+$('.xss').parents().eq(1).find('a').eq(1).click();
+$('[data-action=retweet]').click();
+alert('XSS in Tweetdeck')
+</script> ♥
+```
+
+The first line selects the retweet button and then clicks it, and the second line confirms the action by clicking ok on the modal tht confirms the retweet. A TweetDeck user who saw this tweet will only see "♥", the script portion is executed by the browser on the script. Thus, without the `alert()`, the TweetDeck user will not even notice anything.
+
+You can still see this tweet <a href="https://twitter.com/dergeruhn/status/476764918763749376?lang=en" target="_blank">here</a>.
+
+## How to prevent XSS?
 
 There are a couple of ways to prevent your website against XSS. The two most
 common ways are:
 
-* Escaping String Input: By converting legal HTML characters to their
+**Escaping String Input** <br>
```

Comment:
I don't know if you want to or not (may make it too complex for a new reader?) but you could add warnings that this potentially isn't enough, and thus it is not recommended that it is used as the sole defense mechanism. Just escaping specific black-listed characters has been shown to be insufficient several times. (Google list of naughty strings)